types/color-hash/index.d.ts

```diff
@@ -12,8 +12,8 @@ interface HueObject {
 }
 
 type Hue = number | HueObject | ReadonlyArray<HueObject>;
-type Lightness = number | ColorValueArray;
-type Saturation = number | ColorValueArray;
+type Lightness = number | ReadonlyArray<number>;
+type Saturation = number | ReadonlyArray<number>;
```

Comment:
```suggestion
type Lightness = number | number[];
type Saturation = number | number[];
```
those are props of the options passsed to constructor, not args, not parameters, why readonly?

Reply:
You are absolutely right. Thank you for pointing this out.